crt_portal/static/js/progress-bar.js

```diff
@@ -43,8 +42,9 @@
     }
   } else {
     // if browser doesn't support scrolling and position: sticky, use position: fixed instead
+    var header = document.getElementsByClassName('intake-header')[0];
+    var barOffset = header.getBoundingClientRect().height;
     var bar = document.getElementsByClassName('intake-header--progress-bar')[0];
-    var barOffset = bar.getBoundingClientRect().top;
```

Comment:
if the page was loaded to an anchor tag, this would equal 0, which would have the effect of permanently sticking the progress bar to the top of the window. oops #2